docs/reference-capabilities/default-capabilities.md

```diff
@@ -0,0 +1,233 @@
+# Default Reference Types
+
+We've mentioned throughout this tutorial what the default reference capabilities are for types and classes. However, there's a good chance those defaults passed over your head while you were focused on understanding capabilities themselves.
+
+First, there is nothing saying you have to use default capabilities. Especially as you start writing your first Pony programs, you'll probably find it very educational to explicitly specify the capabilities for all your classes, actors, functions, parameters and fields. However, it's not _writing_ code that makes default capabilities confusing; it's _reading_ that code. Hopefully keeping this summary open will help you navigate pony source code until you've become used to the defaults!
+
+## Reference type summary
```

Comment:
So I think it should be assumed that someone arrives on this page from a search and additional information should be here as well. Before diving into a table, I think there should be a recapping of what a "default capability" is. Something like showing...

```pony
fun foo(): String iso^ =>
  "hello world".clone()
```

and then show the

```pony
fun box foo(): String iso^ =>
  "hello world".clone()
```

Reply:
What about removing the clone and returning `String val` so that you can show the default capability for String?

Reply:
Ah so `String` as the return type in the first and `String val` in the second?

Reply:
Exactly, two birds with one stone that way

Reply:
I created one with a class as well to get even more birds.